### PR TITLE
add expenses feature

### PR DESCRIPTION
**Pull request by MiejkeHoltermann**

### CODE REVIEW

File: components/Header.js [hunk 1]
```diff
@@ -2,32 +2,38 @@ import styled from "styled-components";
...
-            width={40}
-            alt="travel log"
-          />
-        </StyledTravelLogLink>
-        <StyledName>{trip.name}</StyledName>
-        <StyledDate>
-          {format(new Date(trip.startDate), "dd/MM/yy")} -{" "}
-          {format(new Date(trip.endDate), "dd/MM/yy")}
-        </StyledDate>
+        <DefaultButton slug="travellog" icon="book" />
```
Comment (MerleMak) [inline components/Header.js:19]:
```suggestion
        <DefaultButton url="/travellog" icon="book" />
```
You are not receiving a slug in the component, so i guess you mean the url?

File: components/DefaultButton.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import styled from "styled-components";
+import { css } from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
+
+export default function DefaultButton({ url, icon, $orange, children }) {
```
Comment (MerleMak) [inline components/DefaultButton.js:6]:
```suggestion
export default function DefaultLinkButton({ url, icon, $orange, children }) {
```

Comment (MerleMak) [reply components/DefaultButton.js:6]: or just default Link because its a link that just looks like a button ideally :D

**Review by MerleMak — COMMENTED** [review]
I really really like your app! Nice amount of complexity but very readable!
A button can never be a link but a link can look like a button! :) Other than that, no complaints, great job!!
I noticed your input field have different borders, but thats just on the side :D
Great Job Miejke  🚀

File: components/DefaultButton.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                height={60}
+                alt="itinerary"
+              />
+            ) : (
+              <p>{children}</p>
+            )}
+          </StyledLink>
+        </StyledButton>
+      </StyledOptionsLinkBorder>
+    </>
```
[the remark this answers is not in the document]
Comment (MiejkeHoltermann) [reply components/DefaultButton.js:25]: Thank you, I removed the fragments. Actually it's not a button, it's just a div that I styled like a button and placed the link on it. I renamed it anyways to make it a little clearer.

File: components/DefaultButton.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+          width: 6rem;
+          height: 6rem;
+          background: linear-gradient(180deg, #00938c 00%, #006f7a 100%);
+          &:hover {
+            background: linear-gradient(180deg, #006f7a 0%, #00938c 100%);
+          }
+        `}
+`;
+
+const StyledLink = styled(Link)`
```
[the remark this answers is not in the document]
Comment (MiejkeHoltermann) [reply components/DefaultButton.js:80]: Yep, that looks better 😄

**Comment by MiejkeHoltermann** [comment]
Thank you very much for the nice feedback 😊! I implemented your suggestions.

File: components/Header.js [hunk 5]
```diff
@@ -2,9 +2,11 @@ import styled from "styled-components";
 import Image from "next/image";
 import Link from "next/link";
 import { format } from "date-fns";
-import DefaultButton from "@/components/DefaultButton";
+import DefaultLinkButton from "@/components/DefaultLinkButton";
 
 export default function Header({ name, image, startDate, endDate }) {
+  console.log(startDate);
+  console.log(endDate);
```
Comment (MerleMak) [inline components/Header.js:9]:
```suggestion
```

**Review by MerleMak — COMMENTED** [review]
Great job!🚀

**Pull request merged**